bitbots_vision/src/bitbots_vision/dynamic_color_space.py

```diff
@@ -302,8 +302,8 @@ class Heuristic:
     def __init__(self, debug_printer):
         # type: (DebugPrinter) -> None
         """
-        Filters new color space colors according to their position relative to the horizon.
-        Only colors that occur under the horizon and have no occurrences over the horizon get picked.
+        Filters new color space colors according to their position relative to the field_boundary.
```

Comment:
In comments like this where the word "field_boundary" is not used as a specific term it should be written without the under_score.

Reply:
I think that this is not a crucial problem. Let's change these two occurrences and correct others when we see them.

Reply:
'field_boundary' has been changed to 'field boundary' and 'field_boundary-mask' to 'field-boundary-mask' in comments